src/components/document-list/_macro.spec.js

```diff
@@ -4,472 +4,359 @@ import * as cheerio from 'cheerio';
+                );
+                const headingLevel = $('.ons-document-list__item-title')[0].tagName;
+                expect(headingLevel).toBe('h1');
+            });
         });
     });
 
-    describe('mode: with thumbnail', () => {
-        it('passes jest-axe checks', async () => {
+    describe('GIVEN: Params: thumbnail', () => {
+        describe('WHEN: thumbnail is provided for a document', () => {
             const $ = cheerio.load(
                 renderComponent('document-list', {
-                    id: 'some-id',
                     documents: [EXAMPLE_DOCUMENT_LIST_WITH_THUMBNAIL],
                 }),
             );
 
-            const results = await axe($.html());
-            expect(results).toHaveNoViolations();
-        });
-
-        it('has expected `srcset` attribute', () => {
-            const $ = cheerio.load(renderComponent('document-list', { documents: [EXAMPLE_DOCUMENT_LIST_WITH_THUMBNAIL] }));
-
-            const srcset = $('.ons-document-list__image-link img').attr('srcset');
-            expect(srcset).toBe('/example-small.png 1x, /example-large.png 2x');
-        });
-
-        it('has expected `src` attribute', () => {
-            const $ = cheerio.load(renderComponent('document-list', { documents: [EXAMPLE_DOCUMENT_LIST_WITH_THUMBNAIL] }));
-
-            const src = $('.ons-document-list__image-link img').attr('src');
-            expect(src).toBe('/example-small.png');
-        });
-
-        it('has the placeholder class if `thumbnail` is true', () => {
-            const $ = cheerio.load(
-                renderComponent('document-list', {
-                    documents: [{ ...EXAMPLE_DOCUMENT_LIST_BASIC, thumbnail: true }],
-                }),
-            );
-
-            expect($('.ons-document-list__image-link').hasClass('ons-document-list__image-link--placeholder')).toBe(true);
+            test('THEN: passes jest-axe checks', async () => {
+                const results = await axe($.html());
+                expect(results).toHaveNoViolations();
+            });
+
+            test('THEN: has expected srcset attribute', () => {
+                const srcset = $('.ons-document-list__image-link img').attr('srcset');
+                expect(srcset).toBe('/example-small.png 1x, /example-large.png 2x');
+            });
+
+            test('THEN: has expected src attribute', () => {
+                const src = $('.ons-document-list__image-link img').attr('src');
+                expect(src).toBe('/example-small.png');
+            });
+
+            test('THEN: has the right placeholder class', () => {
+                const $ = cheerio.load(
+                    renderComponent('document-list', {
+                        documents: [{ ...EXAMPLE_DOCUMENT_LIST_BASIC, thumbnail: true }],
+                    }),
+                );
+                expect($('.ons-document-list__image-link').hasClass('ons-document-list__image-link--placeholder')).toBe(true);
+            });
```

Comment:
This test should be inside its own when, its testing for when no thumbnail is provided but instead thumbnail is set to true so a placeholder is rendered instead